tests-metadata.json

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Pre-calculated load balancing data. It was done on my personal machine. For purpose of load balancing I don't think it matters that much.